scripts/terraform/deploy.sh

```diff
@@ -43,8 +43,14 @@ ln -s /usr/local/bin/terraform-$TERRAFORM_VERSION /usr/local/bin/terraform
 
 # always init first
 echo "Running terraform init"
-terraform init -input=false
+upgrade=""
+if [[ "$BITOPS_CONFIG_COMMAND" == *"--upgrade"* ]];then
```

Comment:
what do those asterisks do here?

Reply:
why not just have the schema set an environment variable for the `upgrade` option specifically, and then check for that directly rather than try to parse the whole bitops config command?